src/lib/style/components/_download-list.scss

```diff
@@ -68,10 +68,6 @@
   margin-left: 5px;
 }
 
-.btn {
```

Comment:
This was previously `.action-button`. There is another scoped `.action-button` selector that also applies `margin-left: 15px`. I think that the scoped style is applied to every button that this style was originally meant to apply to, so it is redundant.